src/vs/editor/contrib/hover/browser/markdownHoverParticipant.ts

```diff
@@ -212,6 +212,7 @@ class MarkdownRenderedHoverParts extends Disposable {
 
 	private _renderedHoverParts: RenderedHoverPart[];
 	private _hoverFocusInfo: FocusedHoverInfo = { hoverPartIndex: -1, focusRemains: false };
+	private _ongoingHoverOperations: Map<HoverProvider, { verbosityDelta: number; tokenSource: CancellationTokenSource }> = new Map();
```

Comment:
make sure to dispose CTS during object/owner dispose

Reply:
Yes thanks for pointing that out!